hpy/devel/__init__.py

```diff
@@ -10,6 +11,9 @@
 # NOTE: this file is also imported by PyPy tests, so it must be compatible
 # with both Python 2.7 and Python 3.x
 
+DEFAULT_HPY_ABI = 'universal'
```

Comment:
I'm not sure how I feel about the default ABI being different on different Python implementations. It surprised me a bit, and I'm worried it will continue to surprise people.

On the other hand, I can see that on CPython one might want the default to be `cpython` right now and on other Python implementations one would want the default to be something that works well there.

Happy for this to stay this way, just flagging it for thinking about if necessary.

Reply:
I agree with your concerns. Having different defaults is sub-optimal. However, consider that:
1. On CPython, we surely want `cpython` by default, to avoid unwanted slowdowns. I don't want to risk that people starts tweeting that hpy is slow
2. On PyPy, we surely want `universal` by default, because `cpython` does not work at all (or if it does, it's utterly slow)
3. At the moment, people are already used to recompile extensions for every python version they need. So if they just continue to do that, they will get the "best" result on CPython/PyPy/GraalPython.
4. Eventually it will be nice to have `universal` by default, but at the moment it's a bit useless anyway, since we don't have a way to distribute them.